Drive/rover_drive_system.hpp

```diff
@@ -237,6 +237,20 @@ class RoverDriveSystem : public sjsu::common::RoverSystem
   }
 
  private:
+
+  // https://github.com/SJSURoboticsTeam/urc-central-2021/issues/196
```

Comment:
```suggestion
```

Reply:
You can mention the issue in the pull request - not in the code itself as it clutters the code base and makes comments less important